bucket/tortoisesvn.json

```diff
@@ -57,21 +78,28 @@
         ]
     ],
     "post_install": [
-        "$null, $bucket = find_manifest($app)",
-        "$bucketdir = bucketdir($bucket)",
-        "$escdir = $dir -replace \"\\\\\",\"\\\\\"",
-        "(gc \"$bucketdir\\..\\scripts\\tortoisesvn\\tortoisesvn-install.reg.templ\") | % {$_ -replace \"%TSVNHOME%\",\"$escdir\"} | out-file -enc oem \"$dir\\tortoisesvn-install.reg\"",
-        "(gc \"$bucketdir\\..\\scripts\\tortoisesvn\\tortoisesvn-install-tools.reg.templ\") | % {$_ -replace \"%TSVNHOME%\",\"$escdir\"} | out-file -enc oem \"$dir\\tortoisesvn-install-tools.reg\"",
-        "cp \"$bucketdir\\..\\scripts\\tortoisesvn\\tortoisesvn-uninstall.reg\" \"$dir\""
+        "function repl($file) {",
+        "    $esc = $dir.Replace('\\', '\\\\')",
+        "    $it = Get-Item \"$dir\\$file\"",
+        "    ((Get-Content $it.Fullname) -replace '%TSVNHOME%', \"$esc\") | Set-Content \"$dir\\$($it.Basename)\" -Encoding ASCII",
```

Comment:
Now you need COMMON

Reply:
The icon files can be found under `$dir\common`, but for some reason `TortoiseSVN` still needs to load the icons from `Program Files\Common Files`. I previously modified all the location to `Program Files\Common Files`, but found it unnecessary and reverted it back.
 
(It's messed up. Sorry.)

Reply:
Then just replace current manifest with one from my review. There you have updated checkver, autoupdate and it is working.

For the icons i will try again in few hours because I do not know what is hapenning right now.